Fargate doc (#3989)
DataDog/integrations-core: ecs_fargate/README.md

#### Other environment variables
For environment variables available with the Docker Agent container, see the [Docker Agent][15] page. **Note**: Some variables are not be available for Fargate.

For global tagging, it is recommended to use `DD_DOCKER_LABELS_AS_TAGS`. With this method, the Agent pulls in tags from your Docker container labels. This requires you to add the appropriate labels to your other Docker containers. Labels can be added directly in the [task definition][25].

Format for the Agent container:
```shell
{ 
"name": "DD_DOCKER_LABELS_AS_TAGS", 
"value": "{"<LABEL_NAME_TO_COLLECT>":"<TAG_KEY_FOR_DATADOG>"}" 
}
```

Example for the Agent container:
```shell
{ 
"name": "DD_DOCKER_LABELS_AS_TAGS", 
"value": "{"com.docker.compose.service":"service_name"}" 
}
```

> `DD_HOSTNAME` and `DD_TAGS` should not be used since there is no concept of a real host to the user in Fargate. Using these variables may cause problems in your environment.

Reviewer: ```suggestion
**Note**: You shouldn't use `DD_HOSTNAME` and `DD_TAGS` in Fargate since there is no concept of a host.
```

Author: There's no concept of a host to the user. There is a host on the AWS side. I will re-word a little.

> `DD_HOSTNAME` and `DD_TAGS` should not be used since there is no concept of a real host to the user in Fargate. Using these variables may cause problems in your environment.

Author: ```suggestion
**Note**: You should not use `DD_HOSTNAME` and `DD_TAGS` since there is no concept of a host to the user in Fargate.
```
